src/applications/check-in/components/pages/validate/ValidateDisplay.jsx

```diff
@@ -33,41 +33,55 @@ export default function ValidateDisplay({
     [showValidateError],
   );
 
+  useEffect(
+    () => {
+      setDobError(false);
```

Comment:
Shouldn't this go within the `else if` block after line 44? It seems to work ok but It should only get set to false once we verify that it is true and act on it.